Config/DBconfig.php

```diff
@@ -1,7 +1,7 @@
 <?php 
     //Conection for database (MYSQL)
     define('HOST', 'localhost');
-    define('DATABASE', 'gestionMaria');//cambiarlo
+    define('DATABASE', 'pruebas');//cambiarlo
```

Comment:
Si cambias el nombre de la base de datos todos la tendran igual, debes dejarla igual antes de hacer un merge
```suggestion
    define('DATABASE', 'gestionMaria');//cambiarlo
```